cfgov/unprocessed/css/molecules/big-checkbox.less

```diff
@@ -0,0 +1,108 @@
+.m-big-checkbox {
+    margin-bottom: 15px;
+
+    &:last-child {
+        margin-bottom: 30px;
+    }
+    .a-input {
+        @checkbox-width: unit( 18px / @base-font-size-px, em);
+
+        &[type="checkbox"] {
+            .u-visually-hidden();
+
+            .lt-ie9 & {
+                height: 20px;
+                width: 20px;
+                border: 0;
+                float: left;
+                margin: 1em;
+                position: static;
+                width: auto;
+            }
+        }
+
+        &[type="checkbox"] + label {
+            cursor: pointer;
+            margin-bottom: 0;
+        }
+
+        &[type="checkbox"] + label::before {
+            background: white;
+            content: '\a0';
+            display: inline-block;
+            margin-top: unit( 2px / @base-font-size-px, em);
+            margin-right: .6em;
+        }
+
+        &[type="checkbox"] + label::before {
+            height: @checkbox-width;
+            width: @checkbox-width;
+            box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-border;
+        }
+
+        &[type="checkbox"]:checked + label::before {
+            content: @cf-icon-approved;
+            font-family: 'CFPB Minicons';
+            font-size: unit( 16px / @base-font-size-px, em);
+            box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-active;
+            text-align: center;
+        }
+
+        &[type="checkbox"]:hover + label::before {
+            box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-active;
+        }
+
+        &[type="checkbox"]:focus + label::before {
+            box-shadow: 0 0 0 2px @input-active;
+        }
+
+        &[type="checkbox"]:disabled + label {
+            color: @input-border-disabled;
+        }
+
+        &[type="checkbox"]:disabled + label::before {
+            background: @input-bg-disabled;
+            box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-border-disabled;
+            cursor: not-allowed;
+        }
+    }
+
+    label {
+        margin: 0;
+        padding: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em )
+                 unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em )
+                 unit( ( @grid_gutter-width / 1.75 ) / @base-font-size-px, em );
+    }
+
+    &.selected label,
+    &.focus label,
+    input:focus + label,
+    input:checked + label {
+        outline: 1px solid @input-border-focus;
+    }
+
+    &.selected label,
+    input:checked + label {
+        background-color: @input-inset-selected;
+    }
+
+    &__1 {
```

Comment:
That's the approach we had before and the problem is that it gets unwieldy really quickly when you want to have multiple layouts. Right now we just have two, the side-by-side for two and then everything else, but UX can easily ask for another two-column layout. The macros to build this stuff out gets unnecessarily complex. 

The other thing was that the form-l classes weren't ideal for this since checkboxes stack in a different way than complete form fields, so I figured it made sense to bring it into the molecule and treat it as a full-width or half-width modifier.

What do you have in mind?